### ISSUE TITLE
Loss error

### ISSUE THREAD

**Issue by Yaffa1607** [MarvinTeichmann/tensorflow-fcn#13]
Hi ,after building the network on vgg 8 , when I try to run the loss ,it gives me error 
<img width="1124" alt="screen shot 2016-10-25 at 9 48 41 am" src="https://cloud.githubusercontent.com/assets/15791187/19688587/62162ca4-9a98-11e6-8d17-60773ebbe8f5.png">

If I see the shape of vgg.pred_up , it is a question mark. 
It shows values till vgg.score_fr

**Comment by MarvinTeichmann** [comment]
You are calling a module, not the loss function. Try loss.loss()
